ComponentsInitialization.js

```diff
@@ -5,6 +5,10 @@ define(function (require) {
         var MuiThemeProvider = require('material-ui/styles/MuiThemeProvider').default;
         var NetPyNETabs = require('./NetPyNETabs').default;
         var injectTapEventPlugin = require('react-tap-event-plugin');
+        var Console = require('../../js/components/interface/console/Console');
+        var TabbedDrawer = require('../../js/components/interface/drawer/TabbedDrawer');
+        var PythonConsole = require('../../js/components/interface/pythonConsole/PythonConsole');
+        var pythonNotebookPath = "http://" + window.location.hostname + ":" + window.location.port + "/notebooks/notebook.ipynb";
```

Comment:
Something very minor, this variable is used in only one place (once we removed the commented lines). I would just remove the variable and uses inline [here](https://github.com/MetaCell/geppetto-netpyne/pull/58/files#diff-7e494ee04227a4cb5fb4f5766357ab29R25)

Reply:
Will do! Thx